frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -25,6 +25,19 @@ describe("scenarios > question > new", () => {
       cy.contains("Orders").click();
       cy.contains("37.65");
     });
+
+    it("should remove `/notebook` from URL when converting question to SQL/Native", () => {
```

Comment:
This should be `it.skip` so we can merge the test in before the issue is fixed. Also, I think it'd be good to include the issue number, so we can find the repro test easily. Here's [an example](https://github.com/metabase/metabase/pull/12914/files#diff-776068597c6aff5c2add7a9b3615d32aR24).

Reply:
Aaaaah, ok. Now it makes sense. I believe @Opalevanescence mentioned `.skip` in one of her comments, but I obviously didn't connect the dots :). Also, @paulrosenzweig thank you for the example.

I'll fix this and push new changes soon.